lib/components/displayname-cache.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Caches requests in memory and handles expiring them.
+ */
+class RequestCache {
```

Comment:
This file doesn't have a displayname cache, it has a copy/paste of the request cache.

Reply:
This one was meant to be deleted :/